actions/websocket_actions.jsx

```diff
@@ -825,15 +826,24 @@ export async function handleUserUpdatedEvent(msg) {
     const currentUser = getCurrentUser(state);
     const user = msg.data.user;
 
-    if (isGuest(user)) {
+    const config = getConfig(state);
+    const isTimezoneEnabled = config.ExperimentalTimezone === 'true';
+    const userIsGuest = isGuest(user);
+    if (userIsGuest || isTimezoneEnabled) {
```

Comment:
I don't know the background: why is this branching statement only for guests?

Reply:
Im not sure what the background was either but it just so happened I needed the same logic for `getChannelMemberCounts...` that was being done for guest users